library/src/main/kotlin/com/liferay/vulcan/consumer/screens/views/BaseView.kt

```diff
@@ -0,0 +1,29 @@
+package com.liferay.vulcan.consumer.screens.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import com.liferay.vulcan.consumer.model.Thing
+import com.liferay.vulcan.consumer.screens.Action
+import com.liferay.vulcan.consumer.screens.ThingScreenlet
+import com.liferay.vulcan.consumer.screens.ViewModel
+import org.jetbrains.anko.AnkoLogger
+import org.jetbrains.anko.error
+
+/**
+ * @author Alejandro Hernández
+ */
+abstract class BaseView(context: Context, attrs: AttributeSet) :
```

Comment:
This force all screenlets to be containers. Instead of until that could be a view directly... but I don't think we have more solutions